images/build/cross/cloudbuild.yaml

```diff
@@ -37,6 +38,7 @@ substitutions:
   _IMAGE_VERSION: 'v0.0.0-0'
   _PROTOBUF_VERSION: '0.0.0'
   _ETCD_VERSION: 'v0.0.0'
+  _REGISTRY: 'gcr.io/k8s-staging-build-image'
```

Comment:
```suggestion
  _REGISTRY: 'fake.repository/registry-name'
```

Reply:
done, will update the test-infra to add this in the other job